wdio.conf.js

```diff
@@ -42,6 +42,10 @@ const config = {
     selector: '[data-terra-dev-site-content] *:first-child',
   },
 
+  mochaOpts: {
```

Comment:
This isn't something we want to add to the global config. We'll need to look into a different approach to fix the timeout issues you were seeing.